webpack.config.base.js

```diff
@@ -19,8 +19,8 @@ module.exports = (options) => ({
   context: path.resolve(__dirname, 'src'),
   resolve: {
     modules: [
-      path.resolve('./src'),
-      path.resolve('./node_modules')
+      path.resolve(__dirname, 'src'),
+      'node_modules'
```

Comment:
didn't really look into why, but this is the change that was causing the issue, so i reverted it to its previous state